src/core_plugins/kibana/public/management/sections/objects/_objects.js

```diff
@@ -228,11 +229,17 @@ uiModules.get('apps/management')
                     return obj.save({ confirmOverwrite : !overwriteAll });
                   })
                   .catch((err) => {
-                    if (err instanceof SavedObjectNotFound && err.savedObjectType === 'index-pattern') {
-                      conflictedIndexPatterns.push({ obj, doc });
-                      return;
-                    }
+                    if (swallowErrors && err instanceof SavedObjectNotFound) {
+                      if (err.savedObjectType === 'search') {
+                        conflictedSearchDocs.push(doc);
+                        return;
+                      }
 
+                      if (err.savedObjectType === 'index-pattern') {
+                        conflictedIndexPatterns.push({ obj, doc });
+                        return;
+                      }
+                    }
                     // swallow errors here so that the remaining promise chain executes
```

Comment:
This is so confusing now. I get that we want to swallow errors in the original code--so we can continue execution and use `notify`. But the fact that we do that makes passing in a new arg called `swallowErrors` not make much sense. Its purpose is to avoid hitting the `notify` line. Maybe a better name for that arg is `handleConflicts` or `collectDocsNotFound`.